### DISCUSSION TITLE
ERROR: MoviePFMpeg: unsupported codec id '86018"

### DISCUSSION THREAD

**Discussion by SKY123C** [AcademySoftwareFoundation/OpenRV#477] [Q&A] [answered]
ERROR: MoviePFMpeg: unsupported codec id '86018"
hi, how to solve the problem?

**Comment by bernie-laberge** [comment]
This is for the Advanced Audio Codec (AAC) which is disabled by default in the Open RV FFmpeg's default configuration because it is a non free codec.
Provided you are legally entitled to use it, you can simply comment out the following line to enable it:
https://github.com/AcademySoftwareFoundation/OpenRV/blob/0ba6a733689557cc42b925c4bbefcae02fa9d07d/cmake/dependencies/ffmpeg.cmake#L209

I hope this helps.

**Reply by bernie-laberge** [reply, marked as answer]
Note that we are currently thinking of making it easier to do without having to touch any code.
Something like :
`rvcfg RV_FFMPEG_ENABLE_NON_FREE_DECODERS="aac,vp9"`

**Reply by SKY123C** [reply]
thanks

**Reply by loorthu** [reply]
I believe the actual arguments to pass to rvcfg should be

`-DRV_FFMPEG_NON_FREE_DECODERS_TO_ENABLE="aac;vp9"`
